### PR TITLE
Update gacha_records.html

### PR DESCRIPTION
**Pull request by Becods**
Better frontend interface for Gacha records

png

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: data/gacha_records.html [hunk 1]
```diff
@@ -106,7 +106,7 @@ <h2 class="mb-5">Gacha Records</h2>
                     <div class="row">
                         <div class="col-md-6">
                             <span>
-                                Template by <a href="https://preview.tabler.io/">BecodReyes</a>. All rights reserved.
+                                Template by <a href="https://loli.sale/">BecodReyes</a>. All rights reserved.
```
Comment (memetrollsXD) [inline data/gacha_records.html:109]: real

**Comment by memetrollsXD** [comment]
(Please fix your merge conflicts)

**Comment by Becods** [comment]
I didn't know what to do, so I closed it and opened a new one :P

File: data/gacha_records.html (hunk 1 above)
Comment (lilmayofuksu) [reply data/gacha_records.html:109]: real

**Pull request closed without merging**